spring-boot/admin/src/main/resources/application.yml

```diff
@@ -107,3 +125,11 @@ security:
       userInfoUri: https://api.github.com/user
 
 management.security.enabled: false
+
+# Proxy configuration files are stored in this directory. The value of this parameter needs to direct to the same
+# folder as spring.cloud.config.server.native.searchLocations from the Config Servers application.yml.
+proxy.config.files.folder: ${user.dir}/target/temp-gateway-config-file-storage
+
+# If CoatRack is deployed via IDE/commandline the Gateway config files will be stored in the 'target' folder of the
+# project. If docker-compose is used for deployment, these files will be stored internally within a docker volume.
+is-run-via-docker-compose-setup: false
```

Comment:
This is a default yml config that will be put inside the admin jar file. That jar can be run on its own but it could also be put into a container and be run via docker-compose. 

Therefore I think we simply cannot know here if it is running in a docker-compose setup, so we should not define this parameter here. It could lead to unexpected problems in specific deployment situations.

Reply:
To be discussed.

Reply:
I changed the parameter name to `is-gateway-config-files-folder-in-users-home-directory` which is much more fitting than the old name.

You can either run the Admin jar via IDE/CLI which works because of the default value of this parameter in the default profile. Your second option is to run it using the docker-compose.yml which overrides this parameter inverses its value. As far as I know, there are no other deployment options, so where do you see possible problems?

Reply:
If I understand correctly, this is only used to delete the folder in local home? As I would advise against that, see other discussion, maybe we do not need this parameter at all.

The problem I see is simply: this parameter defines something that cannot be known.

Reply:
I changed the overall logic a little bit. So you might want to take another look at it. The docker-compose.yml file provides an environment variable `IS_DEPLOYED_VIA_DOCKER_COMPOSE` for the JVM's of admin and config server. Admin and config-server check if this variable is set or not. Depending on that they 
1. either put the proxy jar and the proxy config files into docker volume folders
2. or use the local project folder to store/find these files.

Reply:
Deprecated.